setup.cfg

```diff
@@ -42,7 +42,7 @@ install_requires =
     terminado>=0.8.3
     prometheus_client
     pywin32>=1.0 ; sys_platform == 'win32'
-    anyio>=2.0.2,<3
+    anyio>=3.0<4
```

Comment:
TODO: Update the lower bound version to the new version with the fix.

Reply:
Also the syntax is incorrect (missing the comma).

Reply:
Ah, good catch. Thanks! I fixed it.